public/components/Notifications/NotificationForm.tsx

```diff
@@ -168,18 +180,26 @@ export const NotificationForm: React.FC<NotificationFormProps> = ({
                   />
                 </EuiCompressedFormRow>
               </EuiFlexItem>
-              {prepareMessage && (
-                <EuiFlexItem>
-                  <EuiCompressedFormRow>
-                    <EuiSmallButton
-                      fullWidth={false}
-                      onClick={() => prepareMessage(true /* updateMessage */)}
-                    >
-                      Generate message
-                    </EuiSmallButton>
-                  </EuiCompressedFormRow>
-                </EuiFlexItem>
-              )}
+
+              <EuiFlexItem>
+                <EuiCompressedCheckbox
+                  id={`checked`}
+                  label={'Preview message'}
+                  checked={displayPreview}
+                  onChange={(e) => onDisplayPreviewChange(e)}
```

Comment:
Here we can just pass `onDisplayPreviewChange` since it already take the event as its argument

Reply:
Done